userguide/drawing_classifier/export-coreml.md

```diff
@@ -167,6 +162,39 @@ func rasterize(drawing stroke_based_drawing:Drawing) -> CGImage {
         space:grayscale, bitmapInfo:CGImageAlphaInfo.none.rawValue)
     let final_rect = CGRect(x: 0.0, y: 0.0, width: 28.0, height: 28.0)
     final_bitmap_context?.draw(intermediate_image!, in: final_rect)
-    return (final_bitmap_context?.makeImage())!
+    return cgImage : (final_bitmap_context?.makeImage())!
+}
+```
+
+
+Through a simple drag and drop process, you can incorporate the model into Xcode. The following Swift code is needed to consume the model in an iOS app.
+
+```swift
+let model = try VNCoreMLModel(for: MySquareTriangleClassifier().model)
+
+let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
+    self?.processClassifications(for: request, error: error)
+
+//Using Bitmap input
+let grayscaleImage = UIImage(named: "MyImage")
+let handler = VNImageRequestHandler(cgImage: grayscaleImage.cgImage!, options: [:])
+
+//Using stroke-based drawing as input
```

Comment:
Yeah, if we're going to have to separate out the two cases via comments here, I think we may as well have two versions of this that are similar but have these differences.